balpy/balpy.py

```diff
@@ -1515,6 +1515,11 @@ def balVaultWeth(self):
 		wethAddress = vault.functions.WETH().call();
 		return(wethAddress);
 
+	def balVaultGetAuthorizer(self):
+		vault = self.balLoadContract("Vault")
```

Comment:
I guess we're not talking about improvements here, but do we need to load the vault contract every time? Wouldn't it make sense to preload it to a `self.vault`?

Reply:
Agreed! We should keep it in mind when we refactor the lib